client/components/userAccount.jsx

```diff
@@ -14,9 +16,15 @@ class UserAccount extends Component {
   }
 
   async componentDidMount() {
-    const { getMovies, getUserPreviousReviews, loggedInUser } = this.props;
+    const {
+      getMovies,
+      getUserPreviousReviews,
+      loggedInUser,
+      adminPreviousOrders,
+    } = this.props;
     await getUserPreviousReviews(loggedInUser.id);
     await getMovies();
+    await adminPreviousOrders();
   }
```

Comment:
lets ditch the async/await here per our conversation with Eliot last night.  Unless for some reason it wasn't working without it?

Reply:
removed